vendor/bat-native-ads/src/bat/ads/internal/notifications.h

```diff
@@ -28,22 +28,24 @@ class Notifications {
 
   bool Get(const std::string& id, NotificationInfo* info) const;
 
-  void Add(const NotificationInfo& info);
+  void PushBack(const NotificationInfo& info);
+  void PopFront(bool should_dismiss);
 
-  bool Remove(const std::string& id);
-  void RemoveAll();
-  void CloseAll() const;
+  bool Remove(const std::string& id, bool should_dismiss);
+  void RemoveAll(bool should_dismiss);
```

Comment:
`should_dismiss` should be `const`

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (11)